bundles/framework/publisher2/handler/AdditionalToolsHandler.js

```diff
@@ -0,0 +1,37 @@
+import { controllerMixin } from 'oskari-ui/util';
+import { ToolPanelHandler } from './ToolPanelHandler';
+import '../../../mapping/layerswipe/publisher/SwipeTool';
+
+class UIHandler extends ToolPanelHandler {
+    constructor (tools, sandbox, consumer) {
+        // ToolPanelHandler adds tools to state so we can reference it here
+        super(tools, consumer);
+        this.sandbox = sandbox;
+        this.updateState({
+            tools: []
+        });
+    };
+
+    init (data) {
+        const hasTools = super.init(data);
+        return hasTools;
+    }
+
+    stop () {
```

Comment:
Would it work if we added stop() to `ToolPanelHandler` and use it directly with a new instance for additional tools without extending it?